composer/tools/copy_environment.py

```diff
@@ -30,20 +30,28 @@
 import argparse
 import ast
 import base64
+import ConfigParser
```

Comment:
ConfigParser was renamed in Python 3. We should use the `six` package to make this script work in Python 3 or Python 2.

`from six.moves import configparser`

https://pythonhosted.org/six/#module-six.moves